### ISSUE TITLE
getmac is not recognized

### ISSUE THREAD

**Issue by guhcostan** [bevry/getmac#36]
ERROR: 'getmac' is not recognized as an internal or external command, operable program or batch file.

The command is not registered as a system variable, it would be interesting for the program to directly access the file path, not the getmac variable.

**Comment by balupton** [comment]
I need more details, as seems it is working fine:

```
> npm i -g getmac
/usr/local/bin/getmac-node -> /usr/local/lib/node_modules/getmac/bin.js
+ getmac@1.4.2
added 5 packages from 15 contributors in 2.049s

> getmac-node 
null '7a:00:8d:b0:b5:01'
```

**Comment by guhcostan** [comment]
Not this, the error occurs when windows does not recognize the 'getmac' command, since it is not configured in the system variables.

The solution would be to execute the command through the file getmac that is in system32.

**Comment by guhcostan** [comment]
![error](https://user-images.githubusercontent.com/22989468/41038209-dd8e3aea-696b-11e8-8aab-bce099a3758b.png)

**Comment by balupton** [comment]
hrmmm, may need to upgrade to [safeps](https://github.com/bevry/safeps) instead

**Comment by balupton** [comment]
however, I guess that folder is where getmac.exe always is right?

**Comment by guhcostan** [comment]
exactly

**Comment by balupton** [comment]
alrighty, want to send a PR? I'll be heading to bed shortly, but can merge and release tomorrow

**Comment by balupton** [comment]
released up to v1.4.3
